Examples/Algorithms/Geant4GeoModel/include/ActsExamples/Geant4GeoModel/GdmlDetectorConstruction.hpp

```diff
@@ -24,6 +26,9 @@ class GdmlDetectorConstruction final : public G4VUserDetectorConstruction {
 
  private:
   std::string m_path;
+
+  /// Construct a GeoModel world volume
+  GeoPhysVol* CreateTheWorld();
```

Comment:
```suggestion
  GeoPhysVol* CreateWorld();
```

Reply:
Could this be `std::unique_ptr`, so it's clear where the ownership is? Does GeoModel retain ownership over the top volume?

Reply:
This is a good question. I will have a look at this and the additional points you have raised below.